zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteRemoteInterpreterManagedProcess.java

```diff
@@ -39,10 +38,10 @@
 /**
  * This class manages start / stop of remote interpreter process
  */
-public class RemoteInterpreterManagedProcess extends RemoteInterpreterProcess
```

Comment:
`RemoteRemoteInterpreterManagedProcess`, What is the meaning of the name of this class? Could it be a little confusing？